test/e2e/specs/specs-i18n/wp-i18n__gutenboarding.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @group i18n
+ */
+
+import {
+	setupHooks,
+	validatePageTranslations,
+	BrowserHelper,
+	DataHelper,
+	GutenboardingFlow,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+const locale = BrowserHelper.getLocale();
```

Comment:
Comparing this to the script in #56940, it looks like we have two general different ways of looping over the locales. 

Here - there's one locale per run execution, so the implication is that we'll need to loop in the bash script in the TeamCity build over all the locales we want to test. We in general haven't done that with any Playwright stuff (we got rid of the massive `run.sh` script), but we certainly could.

Alternatively, in #56940, all of the looping is written programmatically in the script itself, so that you just need to run the script. 

I think of the two approaches, I would prefer the latter where the looping is handled programmatically in the script, but I don't feel too strongly! I just think they should generally match, so we can just run all the I18N tests with the same command in the same TeamCity build.

If it's important with these scripts in this PR to be able to be flexible and to just to run for one locale, maybe we can make the ENV variable be a comma delimited list of locales, and we split and loop over them? That way, you can still just pass one in if you wanted to debug a specific language?

Reply:
I agree that looping should be handled in the TeamCity script. I've already updated #56940 to only [run tests for a single locale](https://github.com/Automattic/wp-calypso/pull/56940/commits/b22ef7c697641b54ad5617ee0355a23fbe57bc0d) using the `LOCALE` env variable.

Reply:
Ack, I'm so sorry, I think that I think I missed the first half of that comment you had left that you changed the locale reading. That makes sense, and sounds good to me - I can go get started on those TeamCity scripts now!